tests/main/snap-userd-socket-activation/task.yaml

```diff
@@ -0,0 +1,47 @@
+summary: Check that root can activate userd via socket activation
+
+systems:
```

Comment:
Could you please add comments explaining why each of those is disabled? For 14.04 it is because there's no systemd in the user session.  About the rest, I'm not sure.